integration-healthcheck/src/main/java/org/symphonyoss/integration/healthcheck/services/PodSessionManagerHealthIndicator.java

```diff
@@ -30,12 +30,16 @@
 public class PodSessionManagerHealthIndicator extends AuthenticationServiceHealthIndicator {
 
   private static final String SERVICE_FIELD = "sessionauth";
+  public static final String FRIENDLY_SERVICE_NAME = "Pod Session Manager";
 
   @Override
   protected ServiceName getServiceName() {
-    return ServiceName.POD_SESSION_MANAGER;
+    return ServiceName.POD;
   }
 
+  @Override
+  protected String getFriendlyServiceName() { return FRIENDLY_SERVICE_NAME; }
```

Comment:
This method should return ServiceName.POD_SESSION_MANAGER.toString(). Please format the code again.

Reply:
Fixed!